examples/crucible.py

```diff
@@ -0,0 +1,228 @@
+import asyncio
+import os
+import textwrap
+import typing as t
+
+import click
+import requests
+from loguru import logger
+from pydantic import StringConstraints
```

Comment:
To answer the question about dependencies, I think we're good to leave this out of pyproject. Although, are there any here we are missing that aren't included in the `examples` extras group?

Reply:
perfect, thanks! ah, my mistake i thought the only one was `requests` (fairly common anyway` but saw it's being used in bandit which makes sense :)